tests, sriov: Do not configure pod network IPv6 on fedora-login

oshoval opened the pull request:
Sriov tests doesn't need to configure ipv6 for pod network,
as they don't check deafault pod network.
Moreoever in case we do need to config it, the right way
will be to use NetworkData, as it would also solve
potential misconfiguration that using hardocoded eth0
might lead to.

Signed-off-by: Or Shoval <oshoval@redhat.com>

```release-note
None
```

oshoval commented:
/cc @EdDev

oshoval commented:
Rebased

EdDev reviewed (changes requested):
Thank you for this cleanup.

>  sriov, Remove unneeded eth0 ipv6 set in sriov tests

tests, sriov: Do not configure IPv6 on fedora-login

> 
> Sriov tests don't check the ipv6 of the pod network,
> hence it's not needed.

How is this related? I do not understand.

> The function that was removed also used hardcoded eth0,
> which is not safe, because the primary network name
> might not be eth0 (depends on the order the interfaces
> were detected during boot sequence).

This is partially correct. The interface name is controlled by cloud-init network settings which overwrites anything that relates to the boot sequence.
I think you can drop this in general, the reason to drop it is because we do not need to the the IPv6 configuration on login, we leave to the cloud-init network.
If you prefer to keep this, just make sure to be accurate to avoid confusion.

oshoval commented:
Addressed comments.

The configuration that was removed, is the default pod network one, that both not needed for sriov tests,
and also doesn't use the right method as you suggested, which is to use NetworkData, instead
using hardcoded eth0, which is prone to errors.
Updated description to reflect that.

EdDev reviewed (approved):
Thanks.

oshoval commented:
Thanks Edy

/cc @qinqon 
can you please take a look ?

AlonaKaplan commented:
/approve

oshoval commented:
Thanks Alona
